docs/variants/protectli_vp66xx/releases.md

```diff
@@ -12,7 +12,44 @@ For details about our release process please read
 </center>
 
 Test results for this platform can be found
-[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
+[here](https://docs.google.com/spreadsheets/
+d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
+
+## v0.9.0 - 2024-08-30
+
+Test results for this release can be found
+[here](https://github.com/Dasharo/osfv-results/blob/main/boards/Protectli/VP66xx/v0.9.0-results.csv)
+
+### Added
+
+- [Initial support for Protectli Alder Lake devices VP66xx](../../unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md)
+- [UEFI Secure Boot support](../../unified-test-documentation/dasharo-security/206-secure-boot.md)
+- [Firmware TPM support](../../unified-test-documentation/dasharo-security/200-tpm-support.md)
+- [Boot logo customization support](../../unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md)
+- [USB boot support](../../unified-test-documentation/dasharo-compatibility/31N-usb-boot.md)
+- [NVMe boot support](../../unified-test-documentation/dasharo-compatibility/312-nvme-support.md)
+- [TPM Measured Boot](../../unified-test-documentation/dasharo-security/203-measured-boot.md)
+- [UEFI Shell](../../unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md)
+- [UEFI Secure Boot](../../unified-test-documentation/dasharo-security/206-secure-boot.md)
+- [Network boot](../../unified-test-documentation/dasharo-compatibility/315b-netboot-utilities.md)
+- [Windows 11 booting](../../unified-test-documentation/dasharo-compatibility/31A-windows-booting.md)
+- [Ubuntu LTS booting](../../unified-test-documentation/dasharo-compatibility/308-debian-stable-and-ubuntu-lts-support.md)
+- [Serial port console redirection](../../unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md#sio004001-serial-port-in-firmware)
+- [Vboot Verified Boot](../../unified-test-documentation/dasharo-security/201-verified-boot.md)
+- [Intel ME soft disable](../../unified-test-documentation/dasharo-security/20F-me-neuter.md)
+- [BIOS flash protection for Vboot recovery region](../../unified-test-documentation/dasharo-security/20J-bios-lock-support.md)
+- [Setup menu password configuration](../../dasharo-menu-docs/overview.md#dasharo-menu-guides)
+- [SMM BIOS write protection](../../dasharo-menu-docs/dasharo-system-features.md#dasharo-security-options)
+- [USB stack disable option in setup menu](../../dasharo-menu-docs/dasharo-system-features.md#usb-configuration)
+- [Network stack disable option in setup menu](../../dasharo-menu-docs/dasharo-system-features.md#networking-options)
+- [Intel ME disable option in setup menu](../../osf-trivia-list/me.md)
+
+## Known Issues
+
+- [Display Port doesn't work](
```

Comment:
Please re-generate the release notes, because the changes do not reflect the review I left in templates

Reply:
Are we generating the release notes file and not filling it manually using yaml files? what review in templates do you mean? Could you link it?

Reply:
I meant this one: https://git.3mdeb.com/dasharo/dasharo/pulls/105

Reply:
Thank you

Reply:
https://github.com/Dasharo/docs/compare/f7f8c669943fb3890306bca06f76e5429e56fd6a..6e81e4b5e7be993be67898a829893a83a5220f9f#diff-659c34c80386e22060961792d547a0b7720de5d79b8a127963a4acdec8d39bfbR48